config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
-  - manager.yaml
+- manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator
```

Comment:
there should not be changes in this file

Reply:
Removed; these changes were made during the `make bundle` command, how can I discern whether or not a change made from there should be included?